### PR TITLE
Changes for running PFEM-DEM problems

### PR DESCRIPTION
**Pull request by AFranci**

### CODE REVIEW

File: applications/swimming_DEM_application/python_scripts/swimming_DEM_algorithm.py
Comment (maceligueta) [inline applications/swimming_DEM_application/python_scripts/swimming_DEM_algorithm.py:102]: Hi @GuillermoCasas . As we spoke, I put this here. However, I hate that this code is a repetition of the function that reads the json file for the dem and for swimming dem. I cannot call the actual code because the DEM solution isn't even created., that's why I copied it here. At least the Json file is read just once.

**Comment by maceligueta** [comment]
I will wait for @GuillermoCasas 's approval.

**Review by GuillermoCasas — CHANGES_REQUESTED** [review]
Mainly ok, but:

1. I think you added some files accidently, like the .info one and others.
2. @maceligueta you could always break that function into even smaller bits and just overwrite the non-repeated ones, as we normally do: It is up to you.

**Comment by AFranci** [comment]
Ok, thanks. I erased some files, please see if now it is ok!

**Comment by maceligueta** [comment]
@GuillermoCasas : we can't call or overwite methods that belong to the DEM solution, because the DEM objects are not created yet.

**Comment by maceligueta** [comment]
Sorry about the mess with the latest commits, I just wanted to delete the file DEM_explicit_solver_var.py.
I think I did it correctly in the end...

**Comment by maceligueta** [comment]
@AFranci , can you check if DEM_explicit_solver_var.py has disappeared from the example?  I can't see this change in the list of changes of this PR.

**Comment by AFranci** [comment]
Now I see it inside
DEMforPFEMcoupledDEMexample_Post_Files_NH_Symplectic_Euler_Dt=5e-05_QuadOrder=2 / InputData

**Comment by AFranci** [comment]
Ok, after the last commit (Deletion of hidden file), it disappeared

**Comment by maceligueta** [comment]
Thanks

**Review by GuillermoCasas — APPROVED** [review]

**Comment by philbucher** [comment]
Can this branch be deleted?

**Pull request merged**
